file_organizer_v2/src/file_organizer/methodologies/para/rules/__init__.py

```
@@ -0,0 +1,36 @@
+    "ConditionEvaluator",
+    "ActionExecutor",
+    "ConflictResolver",
+    "CategoryScorer",
+    "Rule",
+    "RuleCondition",
+    "RuleAction",
+    "EvaluationContext",
+    "RuleMatchResult",
+]
+
+__version__ = "1.0.0"
```

Ruff (0.14.13)

No newline at end of file

Add trailing newline

(W292)